bundles/org.openhab.binding.somfytahoma/src/main/java/org/openhab/binding/somfytahoma/internal/handler/SomfyTahomaBridgeHandler.java

```diff
@@ -514,6 +520,10 @@ private Request sendRequestBuilder(String url, HttpMethod method) {
     }
 
     public void sendCommand(String io, String command, String params) {
+        if (ThingStatus.OFFLINE.equals(thing.getStatus()) && !reLogin()) {
```

Comment:
Because `ThingStatus` is an enum so you can use == in comparisons:

```suggestion
        if (ThingStatus.OFFLINE == thing.getStatus() && !reLogin()) {
```

Reply:
changed